browser/ui/webui/brave_vpn/brave_vpn_localized_string_provider.cc

```diff
@@ -0,0 +1,85 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/ui/webui/brave_vpn/brave_vpn_localized_string_provider.h"
+
+#include "base/no_destructor.h"
+#include "brave/components/l10n/common/localization_util.h"
+#include "components/grit/brave_components_strings.h"
+#include "content/public/browser/url_data_source.h"
+#include "ui/base/webui/web_ui_util.h"
+
+namespace brave_vpn {
+
+void AddLocalizedStrings(content::WebUIDataSource* html_source) {
+  constexpr webui::LocalizedString kLocalizedStrings[] = {
```

Comment:
this needs to be static or the array will be allocated for every call to this function

Reply:
done